server/db/interface.go

```diff
@@ -148,7 +148,7 @@ type OrderArchiver interface {
 	// NewEpochOrder stores a new order with epoch status. Such orders are
 	// pending execution or insertion on a book (standing limit orders with a
 	// remaining unfilled amount).
-	NewEpochOrder(ord order.Order, epochIdx, epochDur int64) error
+	NewEpochOrder(ord order.Order, epochIdx, epochDur int64, epochGap int) error
```

Comment:
```go
// For trade orders, the epoch gap should be db.EpochGapNA, while for cancel orders
// it is the number of epochs since the targeted order was placed, as described in the
// docs for CancelRecord.
```